incubator/zookeeper/Chart.yaml

```diff
@@ -11,3 +11,5 @@ maintainers:
     email: lachlan@deis.com
   - name: Kenneth Owens
     email: owensk@google.com
+  - name: Marc Villacorta
```

Comment:
Are you willing to contribute more and to maintain the chart in the future, since you added yourself here?

Reply:
I plan to add an optional Prometheus exporter but I can not commit to much more.

Reply:
The don't add yourself as owner.